cla_frontend/apps/call_centre/urls.py

```diff
@@ -15,6 +16,8 @@
     ),
     url(r'^admin/', include('call_centre.admin.urls', namespace='admin',)),
 
+    url(r'^zendesk/$', ZendeskView.as_view(), name='zendesk'),
```

Comment:
You need to add ``call_centre_zone_required`` to the view so that only logged-in operators can access it.


    from cla_auth.utils import call_centre_zone_required
    
    url(r'^zendesk/$', call_centre_zone_required(ZendeskView.as_view()), name='zendesk'),